app.js

```diff
@@ -1,10 +1,24 @@
 var express = require('express')
-var app = express()
+var app = express();
+var bodyParser = require('body-parser');
+var mongoose = require('mongoose');
 
-app.get('/', function (req, res) {
-  res.send('Hello World!')
+mongoose.connect('mongodb://localhost/afrokouture');
+
+var db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  //we are connected
+});
+
+app.get('/', function(req, res){
+  res.send('please use api/afrokouture/ ');
+});
+
+app.get('afrokouture/', function(req, res){
```

Comment:
The `/` should be at the beginning of the route, so when your visitor types in your URL it will be something like `https://example.com/afrokouture`. At the moment it would be `https://example.comafrokouture`